src/io.cxx

```diff
@@ -1703,14 +1703,42 @@ namespace ipr
             pp << xpr_decl(d, true);
          }
       };
+
+      struct Location_printer
+      {
+         Printer* pp;
+
+         void operator()(const ipr::Node&) {}
```

Comment:
Should this be ignored or signaled at runtime? If the latter, a comment might help for the next iteration.

Reply:
Added a clarifying comment

```
// Location is only present on nodes derived from ipr::Stmt.
// Nothing to print for other types of nodes at the moment.
```